shell/platform/common/cpp/engine_switches.cc

```diff
@@ -19,6 +19,9 @@ std::vector<std::string> GetSwitchesFromEnvironment() {
   const char* switch_count_key = "FLUTTER_ENGINE_SWITCHES";
   const int kMaxSwitchCount = 50;
   const char* switch_count_string = std::getenv(switch_count_key);
+  if (!switch_count_string) {
+    return switches;
+  }
   int switch_count = std::min(
       kMaxSwitchCount, switch_count_string ? atoi(switch_count_string) : 0);
```

Comment:
The check for `switch_count_string` being `nullptr` could now be removed with the above change. I assume this change was just done for readability? The existing code would work the same AFAICT.

Reply:
Yes, just an early exit for readability. If you don't think it's worth it, happy to just revert.